{
    "artifact_type": "RESOURCE",
    "typeName": "TF::DME::Failover",
    "language": "python37",
    "runtime": "python3.7",
    "entrypoint": "tf_dme_failover.handlers.resource",
    "testEntrypoint": "tf_dme_failover.handlers.test_entrypoint",
    "settings": {
        "version": false,
        "subparser_name": "python37",
        "verbose": 0,
        "force": false,
        "type_name": "TF::DME::Failover",
        "artifact_type": "RESOURCE",
        "use_docker": true,
        "language": "python37",
        "protocolVersion": "2.0.0"
    }
}
